dataactbroker/handlers/fileHandler.py

```diff
@@ -1242,10 +1242,14 @@ def get_submission_comments(submission):
     """
     sess = GlobalDB.db().session
     result = {letter: '' for letter in FILE_TYPE_DICT_LETTER.values() if letter != 'FABS'}
+    result['submission_comment'] = ''
     comments = sess.query(Comment).filter_by(submission_id=submission.submission_id)
     for comment in comments:
-        letter = FILE_TYPE_DICT_LETTER[comment.file_type_id]
-        result[letter] = comment.comment
+        if comment.submission_comment and comment.file_type_id is None:
+            comment_type = 'submission_comment'
+        else:
+            comment_type = FILE_TYPE_DICT_LETTER.get(comment.file_type_id, None)
```

Comment:
couldn't we just do `comment_type = FILE_TYPE_DICT_LETTER.get(comment.file_type_id, 'submission_comment')` ? I'm trying to figure out what other situation we have no file type ID for.

Reply:
That works. The code was based on the `comment.submission_comment` in addition to the none `file_type_id` and can be updated.